backend/api/Pims.Api.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <UserSecretsId>0ef6255f-9ea0-49ec-8c65-c172304b4926</UserSecretsId>
-    <Version>2.2.0-33.24</Version>
+    <Version>2.2.0-33.25</Version>
```

Comment:
Don't want to be stingy with version numbers but wouldn't a hotfix trigger a different change in the version number?

`2.2.1-33.25` vs `2.2.0-33.25`